src/sass/components/_footer.scss

```diff
@@ -0,0 +1,108 @@
+.footer {
+  padding: 0;
+  margin: 0;
+  background-color: white;
+  width: 100%;
+  &__logo {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    flex-direction: column;
+    padding-top: 50px;
+  }
+  &__icon {
+    width: 84.12px;
+    height: 30px;
+  }
+  &__icon--orange {
+    fill: #ff6b09;
+  }
+  &__logo--tel {
+    padding-top: 20px;
+  }
+  &__tel {
+    text-decoration: none;
+    font-weight: 600;
+    font-size: 14px;
+    line-height: 1.21;
+    letter-spacing: 0.08em;
+    color: #111518;
+    padding-top: 20px;
+  }
+  &__img {
+    display: flex;
+    justify-content: center;
+    padding-top: 30px;
+    padding-bottom: 50px;
+  }
+  &__date {
+    height: 70px;
+    background-color: #111518;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+  }
+  &__date--text {
+    font-size: 14px;
+    line-height: 2;
+    color: #b9b9b9;
+    text-decoration: none;
+  }
+}
+.footer__date--text:hover {
+  text-decoration: underline;
+}
+@media screen and (min-width: 768px) {
+  .footer {
+    &__logo {
+      display: flex;
+      padding-top: 0;
+      flex-direction: row;
+      padding: 40px;
+      justify-content: space-between;
+    }
+    &__img {
+      padding: 0;
+      border-left: 1px solid #b9b9b9;
+      padding-left: 24px;
```

Comment:
paddingi można skleić w jeden: padding: 0 0 0 24px; Na ile kojarzę nie jest to błąd, jak wolisz tak rozbite dla czytelności to ok.